csharp_dotnetcore/11.QnAMaker/README.md

```diff
@@ -0,0 +1,50 @@
+﻿This sample shows how to integrate QnA Maker in a simple bot with ASP.Net Core 2 with Application Insights. 
+
+# Concepts introduced in this sample
+The [QnA Maker Service](https://www.qnamaker.a) enables you to build, train and publish a simple question and answer bot based on FAQ URLs, structured documents or editorial content in minutes.
+
+The [Application Insights](https://azure.microsoft.com/en-us/services/application-insights/) enables you to discover actionable insights through application performance management and instant analytics.
+
+In this sample, we demonstrate how to use the QnA Maker service to answer questions based on a FAQ text file as input.
+
+# To try this sample
+
+- Clone the samples repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+```
+
+### Prerequisites
```

Comment:
Weird to have a ### following a #. Consider making this just a heading 2 rather than a heading 3.